### ISSUE TITLE
Is the 747 support planned?

### ISSUE THREAD

**Issue by GEOFS154** [kolos26/GEOFS-LiverySelector#123]
I wanna make a livery for it

**Comment by Egan-31** [comment]
Unfortunately, I guess she's not supported, but we have to wait for the kolos answer to be sure

**Comment by GEOFS154** [comment]
> Unfortunately, I guess she's not supported, but we have to wait for the kolos answer to be sure

Ok, i'll wait

**Comment by BoeingBus747** [comment]
I'm pretty sure iuh has a map for it (just like pretty much everything) but I don't know if it will be supported

**Comment by GEOFS154** [comment]
> I'm pretty sure iuh has a map for it (just like pretty much everything) but I don't know if it will be supported

TXH, i!ll ask him

**Comment by BoeingBus747** [comment]
> > I'm pretty sure iuh has a map for it (just like pretty much everything) but I don't know if it will be supported
> 
> TXH, i'll ask him

He already gave me the map, by lerce ag
![image](https://user-images.githubusercontent.com/114799770/198568937-b12a0fce-eceb-4dd9-8d5d-5bdbc5cf85b5.png)

**Comment by kolos26** [comment]
I have asked for permission to create liveries, but I didn't get it, so unfotunatelly We cannot make loveries for lrx aircrafts.

**Comment by BoeingBus747** [comment]
> I have asked for permission to create liveries, but I didn't get it, so unfotunatelly We cannot make loveries for lrx aircrafts.
Yeah, I talked to him about this. He seems like he doesn't want anyone to paint his aircraft

**Comment by iuhairways** [comment]
i have da map tho 
![texture](https://user-images.githubusercontent.com/103064103/198820260-b52e6882-cce8-48f7-9b86-92e777a6e8bb.png)

**Comment by iuhairways** [comment]
and i migt or might not of made korean air and lufthansa

**Comment by BoeingBus747** [comment]
Did he give you permission to? It seems like he does not want his livs painted

> and i migt or might not of made korean air and lufthansa

**Comment by Ryubunao** [comment]
> Yeah, I talked to him about this. He seems like he doesn't want anyone to paint his aircraft

Idk why he doesn't want to paint his aircraft. And does it include other aircrafts he made like the A320NEO or A330?

**Comment by BoeingBus747** [comment]
> > Yeah, I talked to him about this. He seems like he doesn't want anyone to paint his aircraft
> 
> Idk why he doesn't want to paint his aircraft. And does it include other aircrafts he made like the A320NEO or A330?

Yes, anything with the LERCE AG logo on it

**Comment by iuhairways** [comment]
i think i can paint it as long as lrx doesnt know

**Comment by Egan-31** [comment]
> i think i can paint it as long as lrx doesnt know

Well, make it but keep it in your own pc and then if he allows using his model, you'll be able to share your liveries instantly

**Comment by BoeingBus747** [comment]
> > i think i can paint it as long as lrx doesnt know
> 
> Well, make it but keep it in your own pc and then if he allows using his model, you'll be able to share your liveries instantly

Welp, if LRX finds out about that IUH is gonna be in big trouble

**Comment by Ryubunao** [comment]
> > > i think i can paint it as long as lrx doesnt know
> > 
> > 
> > Well, make it but keep it in your own pc and then if he allows using his model, you'll be able to share your liveries instantly
> 
> Welp, if LRX finds out about that IUH is gonna be in big trouble

rip😕

**Comment by CSN1774** [comment]
> > > i think i can paint it as long as lrx doesnt know
> > 
> > 
> > Well, make it but keep it in your own pc and then if he allows using his model, you'll be able to share your liveries instantly
> 
> Welp, if LRX finds out about that IUH is gonna be in big trouble

LOL
